src/main/java/edu/umich/its/cpm/MigrationController.java

```diff
@@ -371,14 +371,39 @@ public void getProjectSitePages(@PathVariable String site_id,
 	}
 
 	/**
+	 * API call to get CTools site membership
+	 * 
+	 * @return
+	 */
+	@GET
+	@RequestMapping("/siteMembership/{siteId}")
```

Comment:
Totally agree. We need to provide an API list page, like the /direct page in sakai project, to show all the end points available. I will create another jira for this, and for later CPM release.